docs/amended-require-resolution.md

```diff
@@ -0,0 +1,196 @@
+# Amended Require Syntax and Resolution Semantics
+
+## Summary
+
+We need to finalize a syntax for aliases in require statements and determine intuitive resolution semantics as we prepare to build a package management system.
+
+## Motivation
+
+As we prepare to build an official Luau package management system, we need unambiguous alias functionality.
+
+In an [old RFC for require-by-string](https://github.com/luau-lang/luau/pull/969), we removed the `@` prefix from aliases altogether and opted for the following design:
+the path `require("my/module")` is resolved by first checking if `my` is an alias defined in a `.luaurc` file.
+If so, we perform a string substitution on `my`.
+If not, we try to resolve `my/module` relative to the requiring file.
+This is identical to approach (3a) below.
+
+This design changed in the next few months and became what we now have in [require-by-string-aliases.md](require-by-string-aliases.md):
+when requiring an alias, it must be prepended with the `@` prefix.
+Otherwise, we will assume the given string does not contain an alias and will resolve it relative to the requiring file.
+This is identical to approach (1a) below.
+
+There have been disagreements about which approach is more appropriate, so this RFC will serve as a way to document the ongoing discussion and the final decision we make.
+
+## Possible Designs and Drawbacks
+
+Below, we have multiple different options for alias syntax and resolution semantics.
+For each design, assume that the following `.luaurc` file is defined:
+```json
+{
+    "aliases": {
+        "libs": "/My/Libraries/Directory"
+    }
+}
+```
+Additionally, assume that we have a file named `dependency.luau` located in these two locations:
+1. `./libs/dependency.luau`
+2. `/My/Libraries/Directory/dependency.luau`
+
+As part of the push to make require statements more explicit, we also propose removing `paths`, as aliases serve a similar purpose and are more explicit.
+The following options operate under the assumption that `paths` is no longer supported in `.luaurc` files, so we do not need to provide a syntax that corresponds to `paths`.
```

Comment:
I've added a couple of sentences describing the proposal to remove `paths` altogether.